packages/atomizer/bin/atomizer

```diff
@@ -189,6 +160,56 @@ function triggerBuild(state) {
     });
 }
 
+// Setup Atomizer instance
+var programOpts = program.opts();
+var atomizer = new Atomizer({ verbose: !!programOpts.verbose });
+
+// Attempt to load config from `--config` option first, otherwise
+// check for a default config file in the current working directory
+var configFile = programOpts.config;
+if (configFile) {
+    if (!fs.existsSync(configFile)) {
+        console.error('Configuration file ' + chalk.cyan(configFile) + ' not found.');
+        return;
+    }
+    config = require(path.resolve(configFile));
+} else if (fs.existsSync(defaultConfigPath)) {
+    config = require(defaultConfigPath);
+}
+
+// Print help menu if no config is loaded and no arguments are passed in
```

Comment:
Keep the logic from before to show the help screen if no config file is found and no files are passed in